base-theme/assets/css/main.scss

```diff
@@ -80,6 +80,10 @@ h3 {
   color: $black;
 }
 
+a + h3 {
+  margin-top: 1rem;
+}
```

Comment:
I think this is very specific for a particular use-case only. It will select the first `h3` element which is placed immediately after `a (anchor)` elements. Right? I wonder if this will have any side-effect somewhere else?

 I'm also not sure about the issue, as to me, the spacing looks fine, I also had a look into [studio](https://ocw-studio-rc.odl.mit.edu/sites/3-091sc-introduction-to-solid-state-chemistry-fall-2010/type/page/?offset=30) and it looks fine as well. Maybe I'm unable to identify the inconsistency.